svg/elements/feComposite.json

```diff
@@ -295,81 +297,32 @@
               "edge": {
                 "version_added": "≤79"
               },
-              "firefox": {
-                "version_added": null
-              },
-              "firefox_android": {
-                "version_added": null
-              },
-              "ie": {
-                "version_added": null
-              },
-              "opera": {
-                "version_added": false
-              },
-              "opera_android": {
-                "version_added": null
-              },
-              "safari": {
-                "version_added": null
-              },
-              "safari_ios": {
-                "version_added": null
-              },
-              "samsunginternet_android": {
-                "version_added": true
-              },
-              "webview_android": {
-                "version_added": true
-              }
-            },
-            "status": {
-              "experimental": false,
-              "standard_track": true,
-              "deprecated": false
-            }
-          }
-        },
-        "lighterForError": {
```

Comment:
Three things at this point:

- reinstate `lighterForError` (again, sorry for the earlier confusion; I only meant for the `api` equivalent feature to be droppped)
- move `lighterForError` one level deeper, as a child to `operator`
- rename `lighterForError` to `lighter` and rewrite the description to `<code>lighter</code> value`